newIDE/app/src/Utils/Window.js

```diff
@@ -42,16 +84,19 @@ export default class Window {
   }
 
   static setTitleBarColor(newColor: string) {
-    if (electron) {
-      // Nothing to do, the title bar is using the system window management.
-      return;
-    }
-
     if (currentTitleBarColor === newColor) {
       // Avoid potentially expensive DOM query/modification if no changes needed.
       return;
     }
 
+    if (ipcRenderer) {
+      ipcRenderer.invoke('titlebar-set-overlay-options', {
+        color: newColor,
+        symbolColor: '#ffffff',
```

Comment:
what does the symbol represent?
Is there a risk it's visible if it's always white?

Reply:
Good catch. I've added a logic to display the controls in black if the background is dark.